test/integration/services.js

```diff
@@ -23,12 +23,12 @@ describe('app', () => {
           expect($('.services').text()).to.equal('Services at this surgery');
           expect($('.services').next('h3').text()).to.equal('Services patients can book themselves');
           expect($('.self-referrals li').length).to.equal(6);
-          expect($($('.self-referrals li')[0]).text()).to.equal('Asthma clinic');
```

Comment:
Is this reverting the card which was doing the extra service mapping in Mongo? https://trello.com/c/wNToTHWo/289-add-revised-gp-service-list-to-application